tests/console/kdump_and_crash.pm

```diff
@@ -15,12 +15,17 @@ use strict;
 use testapi;
 use utils;
 use kdump_utils;
+use registration;
+use version_utils;
```

Comment:
If you only use one or two function(s) of `version_utils`, you can only import the needed functions like this: `use version_utils qw(is_sle sle_version_at_least);`.